defaults/main.yml

```diff
@@ -53,6 +53,10 @@ postgresql_databases: []
 #   owner: # defaults to postgresql_user
 #   state: # defaults to 'present'
 
+# Privileges to ensure exist
+postgresql_privs: []
+
```

Comment:
Extra empty line. Parameters need to be documented (in README.md file too with example).

Reply:
Added usage examples to the README, removed empty line.

Reply:
Refactoring Readme